buddyboss/buddyboss-platform
# #1772 - fixed: All notifications for Private Group request access are being marked as read once you just visited one of them

### All Submissions:






### Changes proposed in this Pull Request:



Fixes https://github.com/buddyboss/buddyboss-platform/issues/1772 .

### How to test the changes in this Pull Request:

1. Click on the Notification icon on the header
2. See the notifications for the group invitation
3. Click on any Group invitation notification then the notification will be marked as read only clicked notification


### Proof Screenshots or Video
https://vimeo.com/582540478/b169acd9af


### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
